.eslintrc.json

```diff
@@ -2,34 +2,42 @@
   "extends": "google",
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": "2017",
+    "ecmaVersion": 2018,
```

Comment:
what about this? its our own eslint so doesn't matter much but would be nice to know why it changed

Reply:
supposed to be a number